sktime/forecasting/base/_base.py

```diff
@@ -2305,6 +2347,27 @@ def _predict_moving_cutoff(
                 )
         return _format_moving_cutoff_predictions(y_preds, cutoffs)
 
+    def _get_varnames(self, default=None, legacy_interface=True):
+        """Return variable column for DataFrame-like returns.
+
+        Developer note: currently a helper for predict_interval, predict_quantiles,
+        valid only in the univariate case. Can be extended later.
+        """
+        if legacy_interface:
+            var_name = default
+        else:
+            y = self._y
+            if isinstance(y, pd.Series):
```

Comment:
I personally prefer if a function has only one return value or each if-else clause has its own. Perhaps, we can change it as follows:

```python
if isinstance(y, pd.Series):
  return [y.name] if y.name is not None else [0] # <- if y.name is not None could probably also moved to the outer if clause.
elif isinstance(y, pd.DataFrame):
  return y.columns
return [0] # Or raise an exception, unsure if this case should ever happen. However, probably linting requires that line.
```

Reply:
yes, that seems better style